types/jsts/index.d.ts

```diff
@@ -1602,6 +1678,17 @@ declare namespace jsts {
     }
 
     namespace io {
+
+
+        /**
+         * OpenLayers 3 Geometry parser and writer
+         */
+        export class OL3Parser {
+            constructor(geometryFactory?: jsts.geom.GeometryFactory);
+            read(geometry: ol.geom.Geometry): jsts.geom.Geometry;
```

Comment:
You will need to reference the open layer type definition. I'm not a 100% on the syntax, but you can look at how they do it with importing geoson at the top of the [leaftlet](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/leaflet/index.d.ts) type def.

Reply:
Do you think this a good ideia? Since it is just used for integration I was thinking to just define those as 'any'.